test/e2e/framework/framework.go

```diff
@@ -27,19 +27,25 @@ const (
 
 	// Plugin commands
 	AddPluginSource      = "tanzu plugin source add --name %s --type %s --uri %s"
+	UpdatePluginSource   = "tanzu plugin source update %s --type %s --uri %s"
+	ListPluginSources    = "tanzu plugin source list -o json"
 	DeletePluginSource   = "tanzu plugin source delete %s"
 	ListPluginsCmdInJSON = "tanzu plugin list -o json"
 	SearchPluginsCmd     = "tanzu plugin search"
-	InstallPLuginCmd     = "tanzu plugin install %s"
+	InstallPluginCmd     = "tanzu plugin install %s"
+	DescribePluginCmd    = "tanzu plugin describe %s"
 	UninstallPLuginCmd   = "tanzu plugin delete %s --yes"
+	CleanPluginsCmd      = "tanzu plugin clean"
 	JSONOutput           = " -o json"
 	TestPluginsPrefix    = "test-plugin-"
 	PluginSubCommand     = "tanzu %s"
+	PluginKey            = "%s_%s_%s" // Plugins - Name_Target_Versions
 
 	// Central repository
-	CentralRepositoryFeatureFlag         = "features.global.central-repository"
-	CentralRepositoryPreReleaseRepoImage = "TANZU_CLI_PRE_RELEASE_REPO_IMAGE"
-	TanzuCliE2ETestCentralRepositoryURL  = "TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL"
+	CentralRepositoryFeatureFlag             = "features.global.central-repository"
+	CentralRepositoryPreReleaseRepoImage     = "TANZU_CLI_PRE_RELEASE_REPO_IMAGE"
```

Comment:
nit: This might be affected by #105 change. Also, there is a mention in that PR, we should not redefine these variables and use already defined constants.

Reply:
I have defined this to be independent of CLI Core code as its E2E tests. And i have removed setting feature flag.